content/en/docs/concepts/workloads/pods/ephemeral-containers.md

```diff
@@ -9,8 +9,6 @@ weight: 80
 
 <!-- overview -->
 
```

Comment:
```suggestion

{{< feature-state state="stable" for_k8s_version="v1.25" >}}
```
If someone is running an older version of Kubernetes, we want to give them a hint to look at the right docs.